apps/mobile/app/src/main/java/com/glycemicgpt/mobile/presentation/plugin/PluginDetailViewModel.kt

```
@@ -0,0 +1,93 @@
+        }
+
+        collectionJob = viewModelScope.launch {
+            try {
+                flow
+                    .flowOn(Dispatchers.IO)
+                    .conflate()
+                    .collect { descriptor ->
+                        _error.value = null
+                        _detailScreen.value = descriptor
+                    }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)